pkg/apis/camel/v1/maven_types.go

```diff
@@ -43,6 +43,8 @@ type MavenSpec struct {
 	// The Maven build extensions.
 	// See https://maven.apache.org/guides/mini/guide-using-extensions.html.
 	Extension []MavenArtifact `json:"extension,omitempty"`
+	// The CLI options that are appended to the list of arguments for Maven commands.
```

Comment:
Maybe we can include an example of the expected configuration, so, it will be automatically documented in the API

Reply:
Right, good idea, let me add it.

Reply:
I've re-pushed with an example and the link to the Maven CLI options reference.